bin/seed-csv.js

```diff
@@ -0,0 +1,29 @@
+import commander from 'commander';
+import fs from 'fs';
+import csvm from '../app/models/CSVModels.js';
+import {rows} from '../app/models/CSVModels.js';
+
+const { program } = commander;
+
+program
+// cruft templating...
+//  .option('-p, --path <string>', 'path to assets')
+//  .option('-n, --name <string>', 'deck name')
+//  .option('-d, --description <string>', 'deck description')
+  .parse(process.argv);
+
+(async () => {
+    for (var i in rows) {
```

Comment:
This would be `for (const row of rows)`. The `for...of` loop is more commonly used than `for...in`.  If you need to iterate over objects, look into `Object.entries` or `.entries()`, which let you use `for...of` with non-arrays.